### PR TITLE
fix: [oadp-1.2] OADP-2094 Always check restore resource priorities

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Address Tiger's comment https://github.com/openshift/oadp-operator/pull/1132#discussion_r1303365262

### CODE REVIEW

File: tests/e2e/must-gather_suite_test.go
Comment (mateusoliveira43) [inline tests/e2e/must-gather_suite_test.go:1]: this file and `tests/e2e/utils/common_utils.go` were mofified by `go fmt` I think, after running `make test`. I can remove them, if you want

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**